File: src/java/WebNinja/src/test/java/webninja/stepdefinitions/TableConverter.java
package webninja.stepdefinitions;

import cuke4duke.Table;
import webninja.Issue;

import java.util.ArrayList;
import java.util.List;
import java.util.Map;

public class TableConverter {
    public List<Issue> toIssues(Table propertiesList) {
        List<Issue> list = new ArrayList<Issue>();
        for (Map<String, String> properties : propertiesList.hashes()) {

            Issue issue = new Issue(
                    properties.get("Severity").trim(),
                    properties.get("Title").trim());
            list.add(issue);
        }
        return list;
    }
}

File: src/dotnet/WebNinja/WebNinja/features/step_definitions/TableConverter.cs
using System.Collections.Generic;
using Cuke4Nuke.Framework;
using WebNinja.webninja;

namespace WebNinja.features.step_definitions
{
    public static class TableConverter
    {
        public static IList<Issue> ToIssues(this Table propertiesList)
        {
            var list = new List<Issue>();
            foreach (var properties in propertiesList.Hashes())
            {
                var issue = new Issue(
                    properties["Title"].Trim(),
                    properties["Severity"].Trim());
                list.Add(issue);
            }
            return list;
        }
    }
}